spec.bs

```diff
@@ -916,7 +919,9 @@ groups owned by the origin of the current frame, and post-auction updates, which
 of navigator.runAdAuction(), updates the interest groups of all bidder origins that participated in the
 auction.
 
-Both update methods trigger the same algorithm, which is described below, and takes a list of owners to
+<div algorithm="interest group update">

```

Comment:
On line 912 above, can we remove "including move 2 trigger sections out and add updateAdInterestGroup WebIDL" from the TODO as I think your PR covers it?